app/lib/entity.js

```diff
@@ -80,12 +80,12 @@ module.exports = Base => class extends Base {
         return new Promise((resolve) => {
             const listenForPlayers = (data) => {
                 let text = data.toString().trim();
-                let regEx = new RegExp('There are \\d+ of a max of \\d+ players online:');
+                let regEx = /.*There are (\d+) of a max of \d+ players online.*/ig;
```

Comment:
It's looking like the main thing you've changed is the regex for stuff. Why does this work and the previous one didn't?